[transparency-titulus] aggiunta la prima versione del vocabolario

seralf opened the pull request:
Aggiunta la prima versione per il vocabolario `transparency-titulus` nella directory:
`/VocabolariControllati/classifications-for-transparency/transparency-titulus/`

giorgialodi commented:
Ciao @seralf, prima di fare merge, mi sono appuntata alcuni piccoli cambiamenti da fare:

1) si usa questo URI http://spcdata.digitpa.gov.it/Amministrazione/cvtiap ma nella definizione del rightsHolder questo https://w3id.org/italia/data/organization/public-organization/cvtiap Usare solo il primo

2) nella definizione dell'intero vocabolario metterei anche rdfs:label "Controlled Vocabulary for trasparency PA"@en ,  "Vocabolario Controllato Trasparenza PA"@it ;  perché alcuni software usano rdfs:label

3) sono da cambiare tutti i downloadURL di tutte le distribuzioni. Un esempio di modifica necessaria è la seguente: <https://raw.githubusercontent.com/italia/daf-ontologie-vocabolari-controllati/master/VocabolariControllati/classifications-for-transparency.ttl> --> che diventa <https://raw.githubusercontent.com/italia/daf-ontologie-vocabolari-controllati/master/VocabolariControllati/classifications-for-transparency/transparency-titulus.ttl>. 
Bisogna farlo per tutte le distribuzioni

Per il resto del vocabolario benissimo.

Giusto un'ultima nota sul CSV. Per uniformità con altri vocabolari  possiamo usare "codice_1_livello 	label_ITA_1_livello 	label_ENG_1_livello" ecc. 

Fatte le modifiche di cui sopra faccio merge. Grazie mille!

seralf commented:
- [x] fixed: si usa questo URI http://spcdata.digitpa.gov.it/Amministrazione/cvtiap ma nella definizione del rightsHolder questo https://w3id.org/italia/data/organization/public-organization/cvtiap Usare solo il primo

- [x] fixed: nella definizione dell'intero vocabolario metterei anche rdfs:label "Controlled Vocabulary for trasparency PA"@en , "Vocabolario Controllato Trasparenza PA"@it ; perché alcuni software usano rdfs:label

- [x] fixed: sono da cambiare tutti i downloadURL di tutte le distribuzioni. Un esempio di modifica necessaria è la seguente: https://raw.githubusercontent.com/italia/daf-ontologie-vocabolari-controllati/master/VocabolariControllati/classifications-for-transparency.ttl --> che diventa https://raw.githubusercontent.com/italia/daf-ontologie-vocabolari-controllati/master/VocabolariControllati/classifications-for-transparency/transparency-titulus.ttl.
Bisogna farlo per tutte le distribuzioni

- [x] fixed: Per uniformità con altri vocabolari possiamo usare "codice_1_livello label_ITA_1_livello label_ENG_1_livello" ecc.

seralf commented:
idea: potremmo pubblicare anche la query SPARQL usata per produrre il CSV? (in caso la aggiungo alla PR)

giorgialodi commented:
@seralf magari, sì pubblichiamolo. Magari aggiungiamo un readme alla cartella dove spieghiamo il lavoro e aggiungiamo la ( o anche le?) query SPARQL

giorgialodi commented:
ciao @seralf, ho visto il readme e per me va molto bene. Anche le query sparql. Ho visto che hai rivisto alcune voci nel file csv, è giusto? Comunque per me ora il vocabolario va bene e possiamo procedere con il merge. A valle del merge, facciamo due operazioni: 1) pubblicazione del vocabolario nello sparql endpoint 2) comunicazione della prima pubblicazione al gruppo di lavoro nostro interno.

seralf commented:
si le voci le abbiamo (per ora ho fatto direttamente io la commit per brevità) riviste rapidamente perché avevamo notato qualche piccola inconsistenza
1) per lo SPARQL endpoint bisognerà capire come procedere: su questo parliamone a voce
2) grazie!

giorgialodi commented:
per l'1) direi che dobbiamo procedere, ma parliamone sicuramente a voce quanto prima.

seralf commented:
ottimo! grazie

Il giorno mar 22 gen 2019 alle ore 21:38 giorgialodi <
notifications@github.com> ha scritto:

> per l'1) direi che dobbiamo procedere, ma parliamone sicuramente a voce
> quanto prima.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/italia/daf-ontologie-vocabolari-controllati/pull/51#issuecomment-456554256>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFYfFSdg6sb8b23IkPuU0cgSH_VAJScks5vF3a0gaJpZM4aIhSu>
> .
>
